packages/backend-core/src/platform/tenants.ts

```diff
@@ -99,3 +101,19 @@ export async function removeTenant(tenantId: string) {
     throw err
   }
 }
+
+export async function getTenantCreatedAt(tenantId: string) {
+  if (isSelfHostUsingCloud()) {
+    // we have no way of knowing when self-host tenants were created, so we
+    // can't return a createdAt date
+    return undefined
+  }
+
+  return await doInTenant(tenantId, async () => {
+    const settings = await getSettingsConfigDoc()
+    if (settings.createdAt === undefined) {
+      return undefined
+    }
+    return new Date(settings.createdAt)
+  })
```

Comment:
This was the best way I could think of to determine the age of the tenant. The settings config doc appears to be created on install in self-host and at tenant creation in cloud.